docs/book/src/reference/markers/crd.md

```diff
@@ -4,6 +4,6 @@ These markers describe how to construct a custom resource definition from
 a series of Go types and packages.  Generation of the actual validation
 schema is described by the [validation markers](./crd-validation.md).
 
-See [Generating CRDs](/reference/generating-crd.md) for examples.
+See [Generating CRDs](/docs/book/src/reference/generating-crd.md) for examples.
```

Comment:
I don't see this prefix being used elsewhere in docs. I'd say just use relative pathing
```suggestion
See [Generating CRDs](../generating-crd.md) for examples.
```

Reply:
Thanks @estroz , I'll update as suggested.

Reply:
Done 🙂